### ISSUE TITLE
Skus not handled correctly for azure flexibles databases

### ISSUE THREAD

**Issue by dbloquel** [infracost/infracost#2815]
Hello

It looks like infracost is not handeling correctly some Skus on mysq and postgresql flexble databases:
Here is a simple code to show:
Main.tf
```

resource "azurerm_resource_group" "rg" {
  name     = myrg
  location = "francecentral"
}

resource "azurerm_mysql_flexible_server" "mysql-001" {
  name                   = mydb
  resource_group_name    = azurerm_resource_group.rg.name
  location               = "francecentral"
  administrator_login    = "myadmin"
  administrator_password = "myrandompassword"
  sku_name               = "GP_Standard_D2ads_v5"
  version                = "8.0.21"
}

```

Infracost result 
```
`2024-01-08T14:54:44+01:00 INF Enabled policies V2
2024-01-08T14:54:44+01:00 INF Enabled tag policies
Evaluating Terraform directory at .
  ✔ Downloading Terraform modules 
  ✔ Evaluating Terraform directory 
  ⠴ Retrieving cloud prices to calculate costs 2024-01-08T14:54:45+01:00 WRN No products found for azurerm_mysql_flexible_server.mysql-001 Compute (GP_Standard_D2ads_v5), using 0.00
  ✔ Retrieving cloud prices to calculate costs

Project: infra/terraform/mysql_flexible_server

 Name                                         Monthly Qty  Unit            Monthly Cost 
                                                                                        
 azurerm_mysql_flexible_server.mysql-001                                                
 ├─ Compute (GP_Standard_D2ads_v5)                    730  hours                  $0.00 
 ├─ Storage                                            20  GB                     $2.66 
 └─ Additional backup storage               Monthly cost depends on usage: $0.10 per GB 
                                                                                        
 azurerm_private_dns_zone.private_dns_zone                                              
 └─ Hosted zone                                         1  months                 $0.50 
                                                                                        
 OVERALL TOTAL                                                                    $3.16 
──────────────────────────────────
3 cloud resources were detected:
∙ 2 were estimated, 1 of which usage-based costs, see https://infracost.io/usage-file
∙ 1 was free:
  ∙ 1 x azurerm_resource_group

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┓
┃ Project                                            ┃ Monthly cost ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━━┫
┃ infra/terraform/mysql_flexible_server              ┃ $3           ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━━━━━━━━━┛`

```
Here is the problem :
`No products found for azurerm_mysql_flexible_server.mysql-001 Compute (GP_Standard_D2ads_v5), using 0.00`

Note that I have the same problem with azurerm_postgresql_flexible_server and B_Standard_B4ms size.
`  ⠸ Retrieving cloud prices to calculate costs 2024-01-08T14:57:51+01:00 WRN No products found for azurerm_postgresql_flexible_server.postgres_server Compute (B_Standard_B4ms), using 0.00`

**Comment by vdmgolub** [comment]
Hello @dbloquel! Thanks for reporting the issue. We've just merged a fix for this, it will go out in the next release. We'll let you know here when it's live.

**Comment by aliscott** [comment]
@dbloquel thank you 🙏! This has been released in [v0.10.33](https://github.com/infracost/infracost/releases/tag/v0.10.33)
